core/src/main/java/hudson/slaves/JNLPLauncher.java

```diff
@@ -234,19 +234,4 @@ public FormValidation doCheckWebSocket(@QueryParameter boolean webSocket, @Query
 
     }
 
-    /**
-     * Returns true if Java Web Start button should be displayed.
-     * Java Web Start is only supported when the Jenkins server is
-     * running with Java 8.  Earlier Java versions are not supported by Jenkins.
-     * Later Java versions do not support Java Web Start.
-     *
-     * This flag is checked in {@code config.jelly} before displaying the
-     * Java Web Start button.
-     * @return {@code true} if Java Web Start button should be displayed.
-     * @since FIXME
-     */
-    @Restricted(NoExternalUse.class) // Jelly use
-    public boolean isJavaWebStartSupported() {
-        return JavaUtils.isRunningWithJava8OrBelow();
```

Comment:
could probably fix imports

Reply:
Yeah, so does some of the messaging and commands on the node status page. So far I'm trying to get a high-level view of what this might look like and the concerns around doing it.